publish/tests/test_version.py

```diff
@@ -30,6 +31,25 @@ def test_version_from_girder(dandiset_factory, mock_girder_client):
     assert version
 
 
+@pytest.mark.django_db
+def test_version_from_girder_no_metadata(dandiset_factory, mock_girder_client):
+    dandiset = dandiset_factory()
```

Comment:
If you just need a simple instance (with no parameters to the factory), you can use the `dandiset` fixture, [provided by `pytest-factoryboy`](https://pytest-factoryboy.readthedocs.io/en/latest/#model-fixture).

Reply:
Got it. Resolved in https://github.com/dandi/dandi-publish/pull/63/commits/7117a04d5ec021fc59e39c25f3d6539e39fce5b6